Refactor recognizer (#47)
andrerpbts/ruby_marks: README.md

    brew install imagemagick

### Ubuntu

On Ubuntu, the `apt-get` should be enough:

    apt-get install imagemagick libmagickcore-dev libmagickwand-dev

> Note that if you try to install this gem in Ubuntu, then error can happened:

Reviewer: ```suggestion
Note that if you try to install this gem in Ubuntu, this error could happen:
```
